db/migrate/20170223135521_add_type_to_cloud_object_store_object.rb

```diff
@@ -0,0 +1,7 @@
+class AddTypeToCloudObjectStoreObject < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_objects, :type, :string
+    add_index :cloud_object_store_objects, :type
+    add_index :cloud_object_store_objects, [:ems_id, :type]
```

Comment:
Thanks for the index.

Please drop the `:type` index and swap the order of this index

```ruby
add_index :cloud_object_store_objects, [:type, :ems_id]
```